.gitignore

```diff
@@ -17,3 +17,6 @@
 
 # Ignore uploaded files
 /public/uploads
+
+# Ignote simplecov coverage
+/coverage
```

Comment:
oh yeah I didn't have this one, this is the one I got `"trim_trailing_white_space_on_save": true`,  thanks a lot I'll add it to the settings.